src/controlCenter/widgets/volume/sinkInputRow.vala

```diff
@@ -60,3 +65,5 @@ namespace SwayNotificationCenter.Widgets {
         }
     }
 }
+
+
```

Comment:
Nit: remove these extra lines, thanks Uncrustify...

Reply:
Mb, switched code editor and the formatter isn't working properly :upside_down_face: should be fine now :)